commands/webhook.go

```diff
@@ -0,0 +1,472 @@
+	printer.SetSingle(true)
+
+	channelArg, errChannel := command.Flags().GetString("channel")
+	if errChannel != nil || channelArg == "" {
+		return errors.New("Channel is required")
+	}
+	channel := getChannelFromChannelArg(c, channelArg)
+	if channel == nil {
+		return errors.New("Unable to find channel '" + channelArg + "'")
+	}
+
+	userArg, errUser := command.Flags().GetString("user")
+	if errUser != nil || userArg == "" {
+		return errors.New("User is required")
+	}
+	user := getUserFromUserArg(c, userArg)
+	if user == nil {
+		return errors.New("Unable to find user '" + userArg + "'")
+	}
+
+	displayName, _ := command.Flags().GetString("display-name")
+	description, _ := command.Flags().GetString("description")
+	iconURL, _ := command.Flags().GetString("icon")
+	channelLocked, _ := command.Flags().GetBool("lock-to-channel")
+
+	incomingWebhook := &model.IncomingWebhook{
+		ChannelId:     channel.Id,
+		DisplayName:   displayName,
+		Description:   description,
+		IconURL:       iconURL,
+		ChannelLocked: channelLocked,
+		Username:      user.Username,
+	}
+
+	createdIncoming, respIncomingWebhook := c.CreateIncomingWebhook(incomingWebhook)
+	if respIncomingWebhook.Error != nil {
+		return respIncomingWebhook.Error
+	}
+
+	printer.Print("Id: " + createdIncoming.Id)
+	printer.Print("Display Name: " + createdIncoming.DisplayName)
+
+	return nil
+}
+
+func modifyIncomingWebhookCmdF(c client.Client, command *cobra.Command, args []string) error {
+	printer.SetSingle(true)
+
+	if len(args) < 1 {
+		return errors.New("WebhookID is not specified")
+	}
+
+	webhookArg := args[0]
+	oldHook, response := c.GetIncomingWebhook(webhookArg, "")
+	if response.Error != nil {
+		return errors.New("Unable to find webhook '" + webhookArg + "'")
+	}
+
+	updatedHook := oldHook
+
+	channelArg, _ := command.Flags().GetString("channel")
+	if channelArg != "" {
+		channel := getChannelFromChannelArg(c, channelArg)
+		if channel == nil {
+			return errors.New("Unable to find channel '" + channelArg + "'")
+		}
+		updatedHook.ChannelId = channel.Id
+	}
+
+	displayName, _ := command.Flags().GetString("display-name")
+	if displayName != "" {
+		updatedHook.DisplayName = displayName
+	}
+	description, _ := command.Flags().GetString("description")
+	if description != "" {
+		updatedHook.Description = description
+	}
+	iconUrl, _ := command.Flags().GetString("icon")
+	if iconUrl != "" {
+		updatedHook.IconURL = iconUrl
+	}
+	channelLocked, _ := command.Flags().GetBool("lock-to-channel")
+	updatedHook.ChannelLocked = channelLocked
+
+	if _, response := c.UpdateIncomingWebhook(updatedHook); response.Error != nil {
+		return response.Error
+	}
+
+	return nil
+}
+
+func createOutgoingWebhookCmdF(c client.Client, command *cobra.Command, args []string) error {
+	printer.SetSingle(true)
+
+	teamArg, errTeam := command.Flags().GetString("team")
+	if errTeam != nil || teamArg == "" {
+		return errors.New("Team is required")
+	}
```

Comment:
Same `MarkFlagRequired` suggestion can be applied here, and same for the other required arguments